README.md

```diff
@@ -178,6 +178,7 @@ The `Connection` object implements three events related to the reconnecting logi
 | --------------- | ---------- | -------------------------------------------------------------------------------------------------------- |
 | ready           | -          | Fired when authentication is successful and the connection is ready to take commands.                    |
 | disconnected    | -          | Fired when the connection is lost.                                                                       |
+| closed    | -          | Fired when the connection is closed by request(`connection.close()`) or suspend method(`connection.suspend`)                                                                        |
```

Comment:
We shouldn't add events triggered by user interaction. If your code can call `close` or `suspend`, it can also trigger it's `manually closed` handler

Reply:
Many things can be handled by code, for example code can handle of connection retry and other things, But how many costs to maintain and change?
We should do thing the right way and right place. Yes code can fire event manually but if we have multiple place of calling connection close?
I think its good way to listen connection close event.